### PR TITLE
fix: the value of callReplayMax sets zero

### PR DESCRIPTION
**Pull request by coryhh**

### CODE REVIEW

File: arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java [hunk 1]
```diff
@@ -229,6 +229,20 @@ private byte[] getKeyByTargetKey(Map<byte[], Integer> maps, byte[] targetKey) {
...
+
+  private Integer getValueByTargetKey(Map<byte[], Integer> maps, byte[] targetKey,
+      Integer defaultValue) {
+    if (MapUtils.isEmpty(maps)) {
+      return defaultValue;
+    }
+    for (Map.Entry<byte[], Integer> entry : maps.entrySet()) {
+      if (Arrays.equals(entry.getKey(), targetKey)) {
+        return entry.getValue();
+      }
```
Comment (mr3) [inline arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java:241]: Recommended implementing a custom container, ref: `com.google.common.hash.HashCode.BytesHashCode`

Comment (coryhh) [reply arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java]: done

File: arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java [hunk 2]
```diff
@@ -823,4 +806,69 @@ private byte[] createRecordInstanceIdKey(byte[] src) {
   private byte[] createMockKeyWithInstanceIdKey(byte[] src, int index) {
     return CacheKeyUtils.merge(src, index);
   }
+
+  /**
+   * reference:com.google.common.hash.HashCode.BytesHashCode
+   */
+  private static class ByteHashKey {
```
Comment (lucas-myx) [inline arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java:813]: if split an independent class, common use?

File: arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java [hunk 3]
```diff
@@ -202,33 +204,14 @@ private <T extends Mocker> int sequencePutRecordData(MockCategoryType category,
...
-      maps.put(key, count);
-      return count;
-    }
-    count = maps.get(mapKey) + 1;
+  private int updateMapsAndGetCount(Map<ByteHashKey, Integer> maps, byte[] key) {
+    ByteHashKey mapKey = new ByteHashKey(key);
+    int count = maps.getOrDefault(mapKey, 0);
+    count++;
     maps.put(mapKey, count);
     return count;
```
Comment (mr3) [inline arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java:212]: Use `return map.merge(new ByteHashKey(key), 1, Integer::sum)` instead.

File: arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java (hunk 2 above)
Comment (coryhh) [reply arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java]: done

File: arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java (hunk 3 above)
Comment (coryhh) [reply arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java]: done

**Review by lucas-myx — APPROVED** [review]

**Review by mr3 — APPROVED** [review]

**Pull request merged**